account_reconcile_restrict_partner_mismatch/report/account_move_lines_report.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+        <record id="view_account_move_reconciled_report_tree" model="ir.ui.view">
+            <field name="name">Reconciled items with partner mismatch</field>
+            <field name="model">account.reconcile.partner.mismatch.report</field>
+            <field name="arch" type="xml">
+                <tree string="Reconciled items with partner mismatch" create="0">
+                <!-- links are not clickable in tree view 
+                only after open form view -->
+                    <field name="partial_reconcile_id" />
+                    <field name="full_reconcile_id" />
+                    <field name="debit_move_id" />
+                    <field name="debit_amount" />
+                    <field name="debit_partner_id" />
+                    <field name="debit_account_id" />
+                    <field name="debit_account_type_id" />
+                    <field name="credit_move_id" />
+                    <field name="credit_amount" />
+                    <field name="credit_partner_id" />
+                    <field name="credit_account_id" />
+                    <field name="credit_account_type_id" />
+                </tree>
+            </field>
+        </record>
+
+        <record id="action_account_reconcile_partner_mismatch_report" model="ir.actions.act_window">
+            <field name="name">Reconciled items with partner mismatch</field>
+            <field name="res_model">account.reconcile.partner.mismatch.report</field>
+            <field name="view_mode">tree,form</field>
```

Comment:
Since `form` is allowed, shouldn't we define a `form` view with `create="0"` as well ? :thinking: